sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -377,6 +377,11 @@ def create_share(self, **kwargs):
         :keyword bool enable_snapshot_virtual_directory_access:
             Specifies whether the snapshot virtual directory should be accessible at the root of the share
             mount point when NFS is enabled. Default value is True.
+        :keyword bool paid_bursting_enabled: This property enables paid bursting.
+        :keyword int paid_bursting_max_bandwidth_mips: The maximum throughput the file share can support.
```

Comment:
I left a comment to get the typo in generated code fixed, once that if fixed, re-generate and then you can fix the typo in all the keyword inputs.